### DISCUSSION TITLE
https://nvdaes.github.io/nvda-2022-2rc1/

### DISCUSSION THREAD

**Discussion by giscus** [nvdaes/nvdaes.github.io#44] [Announcements]
# https://nvdaes.github.io/nvda-2022-2rc1/

Miércoles, 13 de julio de 2022

https://nvdaes.github.io/nvda-2022-2rc1/

**Comment by nvdaes** [comment]
En este post empezamos a usar Giscus para dejar comentarios. Así no será necesario crear incidencias para cada entrada del blog.
